install-moveit2/source/index.markdown

```diff
@@ -26,7 +26,7 @@ These instructions assume you are running on Ubuntu 22.04 (Humble, Rolling), or
 
 ### Install <img src="/assets/install_page/ros_logo.jpeg"/>
 
-Install ROS 2 [Foxy](https://docs.ros.org/en/foxy/Installation/Ubuntu-Install-Debians.html), [Galactic](https://docs.ros.org/en/galactic/Installation/Ubuntu-Install-Debians.html), [Humble](https://docs.ros.org/en/humble/Installation/Ubuntu-Install-Debians.html), or [Rolling](https://docs.ros.org/en/rolling/Installation/Ubuntu-Install-Debians.html) following the installation instructions. We recommend Humble for stable latest LTS distribution needs, and Rolling for contributing to MoveIt 2. Currently the [main branch](https://github.com/ros-planning/moveit2) of MoveIt 2 is supported on both Rolling and also Humble, but since it's used for latest development, it's unstable. For stable versions, please use the distro branches [foxy](https://github.com/ros-planning/moveit2/tree/foxy), [galactic](https://github.com/ros-planning/moveit2/tree/galactic), or [humble](https://github.com/ros-planning/moveit2/tree/humble).
+Install ROS 2 [Humble](https://docs.ros.org/en/humble/Installation/Ubuntu-Install-Debians.html), [Iron](https://docs.ros.org/en/iron/Installation/Ubuntu-Install-Debians.html), or [Rolling](https://docs.ros.org/en/rolling/Installation/Ubuntu-Install-Debians.html) following the installation instructions. We recommend Humble for stable latest LTS distribution needs, and Rolling for contributing to MoveIt 2. Currently the [main branch](https://github.com/ros-planning/moveit2) of MoveIt 2 is supported on both Rolling and also Humble, but since it's used for latest development, it's unstable. For stable versions, please use the distro branches [foxy](https://github.com/ros-planning/moveit2/tree/foxy), [galactic](https://github.com/ros-planning/moveit2/tree/galactic), or [humble](https://github.com/ros-planning/moveit2/tree/humble).
```

Comment:
```suggestion
Install ROS 2 [Humble](https://docs.ros.org/en/humble/Installation/Ubuntu-Install-Debians.html), [Iron](https://docs.ros.org/en/iron/Installation/Ubuntu-Install-Debians.html), or [Rolling](https://docs.ros.org/en/rolling/Installation/Ubuntu-Install-Debians.html) following the installation instructions. We recommend Humble for stable latest LTS distribution needs, and Rolling for contributing to MoveIt 2. Currently the [main branch](https://github.com/ros-planning/moveit2) supports Rolling, Iron, and Humble. However, since it's used for latest development, the main branch is unstable. For stable versions, please use the distro branches [humble](https://github.com/ros-planning/moveit2/tree/humble), or [iron](https://github.com/ros-planning/moveit2/tree/iron).
```
Let's drop foxy and galactic here as well.

Reply:
(I've just created the iron branch from main)

Reply:
and what about main page? In the Current and Future Versions section there are also

Reply:
we should either remove them or call them "EOL - discontinued"